tasks/install.yml

```diff
@@ -18,11 +18,24 @@
 
 - name: Enable PowerTools repo
   # NB: doesn't run command  `dnf config-manager --set-enabled PowerTools` as can't make that idempotent
-  lineinfile:
+  community.general.ini_file:
     path: /etc/yum.repos.d/Rocky-PowerTools.repo
-    create: false # raises error if not already installed
-    regexp: enabled=
-    line: enabled=1
+    section: powertools
+    option: enabled
+    value: "1"
+    create: false
+    no_extra_spaces: true
+  when: ansible_distribution_major_version == '8'
+
+- name: Enable CRB repo
+  community.general.ini_file:
```

Comment:
I think `yum_repository` should be able to handle this sort of thing directly: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_repository_module.html

Reply:
Can you explain? I think I'd have to copy all the things like baseurl, mirror list etc from the existing repo into an `ansible.builtin.yum_repository` task? Which seems less clear/more prone to go wrong than saying "in this existing repofile, change enabled to true"?